### PR TITLE
Enhance About Us Section

### LINKED ISSUE #41: Improve About Us section
Right now, the about us section right below the main logo section is plain. Improve it with more specifics about what makes us special.

* In house curriculum
* Code clubs bringing together girls from multiple schools, giving them a new friend group
* Building upon the curriculum with the girls for multiple weeks / sessions
* Volunteer ran
* Free for girls

### PR DESCRIPTION
**Pull request by troynguyen8**
Enhanced About Us section by:
* adding link in nav to "about us" section.
* adding icons and metrics to measure our impact
* rephrasing our description, including subtext about a few things that make us unique

Changed Nav by:
* increasing contrast. There is no longer transparency on the links. Instead, they are white by default, and on hover they are grey.
* Changing the expand icon on screen widths to be white. Changed the "hamburger" icon to use font awesome instead of default SVG specified in bootstrap

Changed Our Clubs Section:
* Added subtext about what our clubs are

Desktop:
image

Tablet (iPad):
image

Mobile (iPhone X):
image

closes #41

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -83,59 +86,107 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
-          girls.
-        </p>
+      <section id="about-us" class="col-12 justify-content-center bg-grey py-5">
+        <div class="col-xs-10 col-md-8 text-center mx-auto mb-5">
+          <h3 class="mb-3 font-weight-bold">
+            Girls Code Lincoln is a 501(c)(3) nonprofit organization. We strive to ignite passion for technology and leadership in young
+            girls with the long-term goal of closing the gender gap in S.T.E.M.
+          </h3>
+          <i>
+            We pride ourselves in partnering with Nebraska organizations to provide multiweek technology clubs free of charge.
```
Comment (justinjstark) [inline src/index.html]: Use active voice, put what we do first. Maybe something like: "We host multiweek technology clubs for 4th--9th grade girls. All clubs are completely free and 100% volunteer-run. We partner with Nebraska organizations and build in-house curriculum to teach technology, leadership, and life skills."

File: src/index.html [hunk 2]
```diff
@@ -83,59 +86,107 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
+            <h3 class="impact-figure font-weight-bold">200+</h3>
+            <h5 class="impact-subtext">Girls Impacted</h5>
+          </div>
+          <div class="col-xs-6 col-sm-2 d-flex flex-column align-items-center text-center mt-3 mt-sm-0">
+            <span class="fa-stack fa-4x">
+              <i class="fas fa-circle fa-stack-2x icon-background"></i>
+              <i class="fas fa-sync-alt fa-stack-1x icon-foreground"></i>
+            </span>
+            <h3 class="impact-figure font-weight-bold">59%</h3>
+            <h5 class="impact-subtext">Girls Participated for 2 Semesters or More</h5>
```
Comment (justinjstark) [inline src/index.html]: Spell out "Two"

File: src/index.html [hunk 3]
```diff
@@ -83,59 +86,107 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
+        </div>
       </section>
 
       <section id="our-clubs" class="bg-pink col-12 py-5">
-        <h2 class="display3 text-center mb-5 font-weight-bold">Our Clubs</h2>
+        <div class="text-center col-lg-7 col-xs-8 mx-auto mb-3">
+          <h2 class="font-weight-bold">Our Clubs</h2>
+          <i>
+            Our clubs give girls interested in coding a gentle introduction to software, hardware,
+            computer science fundamentals, and leadership through projects, activities, and community speakers
```
Comment (justinjstark) [inline src/index.html]: Add a period at the end

**Comment by troynguyen8** [comment]
@justinjstark How does it look now? It's worth noting that the navbar does odd text wrapping currently with screen widths in the range of 730 - 770 px (notably effects the original iPad and modern iPad minis). I will add a fix for this in #47 via a priority nav, since we are only going to add more navigation links.

**Pull request merged**